tasks/src/text-to-video/data.ts

```diff
@@ -36,7 +36,24 @@ const taskData: TaskDataCustom = {
 		outputs: [],
 	},
 	isPlaceholder: true,
-	metrics:       [],
+	metrics:       [
+		{
+			description: "Inception Score uses an image classification model that predicts class labels and evaluates how distinct and diverse the images are. A higher score indicates better video generation.",
+			id: "is",
+		},
+		{
+			description: "Frechet Inception Distance uses last activations before the classifier layer of an image classification model as embeddings. The metric compares mean and standard deviation of the embeddings of real and generated images. A smaller score indicates better video generation.",
```

Comment:
Is it ok if you could use less internal technical details? This guide is directed for people who don't have a lot of deep technical knowledge.

Reply:
I guess i went a little overboard with the details you'd asked before hehe, I tried to simplify it a bit :)